src/service/evm.rs

```diff
@@ -465,14 +474,16 @@ where
 		transaction_pool: transaction_pool.clone(),
 		task_manager: &mut task_manager,
 		config: parachain_config,
-		keystore: params.keystore_container.sync_keystore(),
+		keystore: params.keystore_container.keystore(),
 		backend: backend.clone(),
 		network: network.clone(),
 		system_rpc_tx,
 		tx_handler_controller,
+		sync_service: sync_service.clone(),
 		telemetry: telemetry.as_mut(),
 	})?;
 
+	// do we need this at all?
```

Comment:
**CLARIFICATION**

@wischli What makes think we do not need them?

Reply:
I think this comment belonged to codepath which I eventually removed. Will remove the comment on Monday.